### PR TITLE
fix exit code

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: FrontBundle/Command/OrchestraGenerateRobotsCommand.php [hunk 1]
```diff
@@ -41,7 +41,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $this->generateRobots($site, $output);
             } else {
                 $output->writeln("<error>No website found with siteId " . $siteId . ".</error>");
-                return 0;
+                return 1;
```
Comment (nicolasThal) [inline FrontBundle/Command/OrchestraGenerateRobotsCommand.php:45]: can you put an empty line before the return

**Comment by ngilain** [comment]
:+1:

File: FrontBundle/Command/OrchestraGenerateRobotsCommand.php [hunk 2]
```diff
@@ -53,7 +53,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $output->writeln("<info>Done.</info>");
-        return 1;
+        return 0;
```
Comment (nicolasThal) [inline FrontBundle/Command/OrchestraGenerateRobotsCommand.php:58]: can you put an empty line before the return

File: FrontBundle/Command/OrchestraGenerateSitemapCommand.php [hunk 3]
```diff
@@ -37,7 +37,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $this->generateSitemap($site, $output);
             } else {
                 $output->writeln("<error>No website found with siteId " . $siteId . ".</error>");
-                return 0;
+                return 1;
```
Comment (nicolasThal) [inline FrontBundle/Command/OrchestraGenerateSitemapCommand.php:41]: same

File: FrontBundle/Command/OrchestraGenerateSitemapCommand.php [hunk 4]
```diff
@@ -49,7 +49,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $output->writeln("<info>Done.</info>");
-        return 1;
+        return 0;
```
Comment (nicolasThal) [inline FrontBundle/Command/OrchestraGenerateSitemapCommand.php:54]: same

**Comment by nicolasThal** [comment]
Does the command works now?

If you return a code 1, does the deploy task stops and rollback?

**Comment by alavieille** [comment]
yes it works, 
with capistrano if code is different of 0, the deploy task is aborded but rollback i don't know

@nicolasThal done for empty line

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**